### PR TITLE
Emit owner at deployment

### PR DESCRIPTION
**Pull request by MerlinEgalite**
Fixes https://github.com/cantinasec/review-morpho-blue-1/issues/34

### CODE REVIEW

File: src/Morpho.sol [hunk 1]
```diff
@@ -67,8 +67,10 @@ contract Morpho is IMorpho {
     constructor(address newOwner) {
         require(newOwner != address(0), ErrorsLib.ZERO_ADDRESS);
 
-        owner = newOwner;
         DOMAIN_SEPARATOR = keccak256(abi.encode(DOMAIN_TYPEHASH, block.chainid, address(this)));
+        owner = newOwner;
+
+        emit EventsLib.SetOwner(newOwner);
```
Comment (Rubilmax) [inline src/Morpho.sol:73]: I would internalize it in a function... But @MathisGD @QGarchery will be against?

**Review by Rubilmax — APPROVED** [review]
Internalizing the storage update and the event emitted would have prevented this, the same way using a third-party `Ownable` contract would have... I don't get why we're this stubborn

**Review by Jean-Grimal — APPROVED** [review]

**Comment by MerlinEgalite** [comment]
@QGarchery can you have a look please?

**Comment by QGarchery** [comment]
> I don't get why we're this stubborn

It could be said that suggesting the same thing multiple times is the subborn behavior :wink: 

The refactor you are suggesting can lead to unexpected side-effects too, like in [this issue](https://github.com/morpho-org/metamorpho/issues/221). There are trade-offs to different implementations, which fits different needs, but let's go into the details again

**Pull request merged**